inc/woocommerce.php

```diff
@@ -73,3 +73,39 @@ function primer_woo_shop_layout( $layout ) {
 
 }
 add_filter( 'theme_mod_layout', 'primer_woo_shop_layout' );
+
+/**
+ * Filter the page title for the WooCommerce shop page.
+ *
+ * @param  string $title
+ *
+ * @return string
+ */
+function primer_woo_shop_title( $title ) {
+
+	if ( function_exists( 'wc_get_page_id' ) && function_exists( 'is_shop' ) && is_shop() ) {
+
+		$title = get_the_title( wc_get_page_id( 'shop' ) );
+
+		// Remove the WooCommerce shop loop title
```

Comment:
@EvanHerman once we are using the `__return_null` callback, this comment won't be necessary.